domino-ui/src/main/java/org/dominokit/domino/ui/dropdown/MenuNavigation.java

```diff
@@ -24,7 +24,7 @@ public MenuNavigation(List<V> items) {
         this.items = items;
     }
 
-    public static <V extends IsElement> MenuNavigation<V> create(List<V> items) {
+    public static <V extends IsElement<E>, E extends HTMLElement> MenuNavigation<V> create(List<V> items) {
```

Comment:
can we make this `public static <V extends IsElement<?>> MenuNavigation<V> create(List<V> items) {`